dashboard/src2/pages/LoginSignup.vue

```diff
@@ -592,6 +638,23 @@ export default {
 				},
 			};
 		},
+		recover2FA() {
+			return {
+				url: 'press.api.account.recover_2fa',
+				onSuccess: () => {
+					toast.success('2FA recovered successfully');
```

Comment:
```suggestion
					toast.success('2FA reset successfully. Please re-enable it as soon as possible!');
```

or something along the lines to nudge the user to enable it right after login